ci/test/pipeline.template.yml

```diff
@@ -351,7 +351,7 @@ steps:
   - id: ssh-connection
     label: SSH connection tests
     depends_on: build-x86_64
-    timeout_in_minutes: 20
+    timeout_in_minutes: 30
```

Comment:
Does this test routinely take more than 20m!? That seems like a lot of $$ to burn on testing SSH tunnels.

Reply:
yeah...we have lots a weird edge cases we have to cover...we could break this into kafka and pg mzcompose's if need be

Reply:
I have some ideas to get it back down to 20min